src/mlpack/methods/ann/layer/minibatch_discrimination.hpp

```diff
@@ -0,0 +1,190 @@
+/**
+ * @file minibatch_discrimination.hpp
+ * @author Saksham Bansal
+ *
+ * Definition of the MinibatchDiscrimination layer class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_MINIBATCH_DISCRIMINATION_HPP
+#define MLPACK_METHODS_ANN_LAYER_MINIBATCH_DISCRIMINATION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+#include "layer_types.hpp"
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Implementation of the MinibatchDiscrimination layer class. The
+ * MinibatchDiscrimination class is a layer of the discriminator that allows
```

Comment:
Please change the above two lines to `Implementation of the MiniBatchDiscrimination layer. MinibatchDiscrimination is a layer ...`